engine/src/main/java/org/terasology/math/MatrixUtils.java

```diff
@@ -94,6 +94,10 @@ public static FloatBuffer matrixToFloatBuffer(Matrix3f m, FloatBuffer fb) {
         return fb;
     }
 
+    public static FloatBuffer matrixToFloatBuffer(Matrix3fc m, FloatBuffer fb) {
```

Comment:
What exactly does this do (aka, please add Javadoc)? 

My guess is taht the matrix `m` will stay unchanged, while the buffer `fb` will be filled with the matrix values?